src/stores/AccountsStore.js

```diff
@@ -361,7 +361,6 @@ export default class AccountsStore extends Container<AccountsStoreState> {
 		// current encryption with only seedPhrase is compatible.
 		updatedIdentity.encryptedSeed = await encryptData(seedPhrase, pin);
 		//TODO now hard coded to polkadot canary prefix which is 2, future enable user to change that.
-		await this._addPathToIdentity('', seedPhrase, updatedIdentity, 'Root', 2);
```

Comment:
can you explain this change?  does it address the TODO and if so, should the TODO be deleted as well?

Reply:
In the previous version we automatically create a root account without network affiliation if user create a new identity, this line of code does that.

In the current version we only allow user to explicitly create a root account with a network key chosen.

And yes, the TODO need to be deleted.